modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -224,70 +225,89 @@ def get_filezilla_creds(paths)
 
       # Assemble the options hash for creating the Metasploit::Credential::Login object
       login_data ={
-          core: credential_core,
-          status: Metasploit::Model::Login::Status::UNTRIED
+        core: credential_core,
+        status: Metasploit::Model::Login::Status::UNTRIED
       }
 
       # Merge in the service data and create our Login
       login_data.merge!(service_data)
       login = create_credential_login(login_data)
-
     end
 
+    vprint_status("       FTP Port: %s" % config['ftp_port'])
+    vprint_status("    FTP Bind IP: %s" % config['ftp_bindip'])
+    vprint_status("            SSL: %s" % config['ssl'])
+    vprint_status("     Admin Port: %s" % config['admin_port'])
+    vprint_status("  Admin Bind IP: %s" % config['admin_bindip'])
+    vprint_status("     Admin Pass: %s" % config['admin_pass'])
+    vprint_line("")
```

Comment:
@g0tmi1k +1: Empty verbose line is sort of difficult to imagine at first but in this case it makes complete sense.